data/strategies/StrategyMakerLINKDAIDelegate.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "strategy",
+  "name": "StrategyMakerLINKDAIDelegate",
+  "description": "Locks LINK at MakerDAO via LINK-A to mint DAI. The newly minted DAI is then deposited into the v2 DAI yVault to earn yield.",
```

Comment:
Do we want to say something about the debt ratio? or may tiny bit longer explanation of this strat?